tasks/getHistoricalData.ts

```diff
@@ -91,6 +91,12 @@ task("getHistoricalData", "Retrieves the historical rates")
     "ETH",
     types.string
   )
+  .addOptionalParam(
```

Comment:
Was this copied from Sol's code? May now be out of date cos I think he was going to switch to a flag.

Reply:
@ioanaroceanu2000 i think @Sol3iman mentioned that you had a catchup with him on how best to align your code with his, worth rebasing his changes on top of yours as Sol's code is now merged into main and make sure there is alignment